meeting_minutes/220714_SARIF_TC_62.md

```diff
@@ -0,0 +1,157 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Member                  |
+| Aharon     | Abadi     | WhiteSource       | Voting Member           |
+| Chris      | Meyer     | Microsoft         | Voting Member           |
+| Chris      | Wysopal   | Veracode          | Member                  |
+| David      | Keaton    | Individual        | Chair                   |
+| Jeff       | Williams  | Contrast Security | Voting Member           |
+| Johnathan  | Gilday    | Contrast Security | Member                  |
+| Mary       | Martin    | Microsoft         | Member                  |
+| Michael    | Fanning   | Microsoft         | Voting member           |
+| Paul       | Anderson  | Grammatech, Inc.  | Voting member           |
+| Pavel      | Mikula    | Sonar             | Member                  |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member           |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member           |
+| Yossi      | Banai     | Microsoft         | Member                  |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70160/agenda_20220714.html
+
+Agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-06-30_Meeting #61](https://www.oasis-open.org/committees/document.php?document_id=70117&wg_abbrev=sarif)
+
+Minutes were approved.
+
+## 1.6 Review of action items and resolutions (Co-Chair David)
+
+* ACTION on Michael to initiate contact with OpenSSF
+  * COMPLETE
+* ACTION on Chris M. to solicit a working example from WhiteSource of their scenario
+  * ONGOING
+ 
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Aharon Abadi will obtain voting rights at the end of this meeting.
+* Johnathan Gilday will obtain voting rights at the end of this meeting.
```

Comment:
These two belong in section 1.7.4 with Aditya.  Aharon, Jonoathan, and Aditya all previously lost voting rights, and this is their first meeting back since then.

Reply:
Sorry, a correction to my correction.  Jonathan does not fit in section 1.7 because he attended one meeting after becoming a full member, but then skipped a meeting.  Aharon still should be moved to section 1.7.4 (he gains voting rights at the end of next meeting if he attends it, not this meeting).

Reply:
I hope that I got that right (now)